src/features/profile/components/MyProfileGeneral.tsx

```diff
@@ -23,6 +25,8 @@ const ProfileGeneral = ({
   isEmpty,
   nativeCurrency,
 }: TProfileGeneral): JSX.Element => {
+  const { locations } = useAppSelector(state => state.pastelProfile)
```

Comment:
NB: Better to use
```ts
  const locations = useAppSelector(state => state.pastelProfile.locations)
```
At the moment it's not a problem, but later pastelProfile may contain a lot of fields and this component will re-reneder on any field change.
When accessing value in `useAppSelector` component will re-render only on specified field change

Reply:
fixed